ui/Filters.elm

```diff
@@ -0,0 +1,1030 @@
+module Filters exposing (..)
+
+
+products : List String
+products =
+    [ "firefox", "thunderbid", "fennec" ]
+
+
+channels : List String
+channels =
+    [ "aurora", "beta", "nightly", "release" ]
+
+
+platforms : List String
+platforms =
+    [ "linux-i686"
```

Comment:
You are missing the android ones.
![builds](https://cloud.githubusercontent.com/assets/229453/26631176/2e3f4500-460a-11e7-8343-a670a94da431.png)